### PR TITLE
docs: recommend users install the GitHub or Bitbucket Cloud hosted app

### LINKED ISSUE #29372: Add Mend app guide to the top of Bitbucket Cloud readme
### Describe the proposed change(s).

The easiest way to get started on Bitbucket Cloud is now using the [Mend app](https://marketplace.atlassian.com/apps/1232072/mend?tab=overview&hosting=cloud) with free Renovate plan. But if you search for something like "how to get started with Renovate for Bitbucket Cloud" you find https://docs.renovatebot.com/modules/platform/bitbucket/ without any mention of it.

Ideally docs.mend.io should have a perfect landing page for how to use the free Renovate. The only current docs page there ([link](https://docs.mend.io/bundle/platform/page/install_mend_developer_platform_for_bitbucket_cloud.html)) is too focused on Mend's paid options and can confuse those who want a free plan. We should add a page there for how to get started with Renovate in Mend, but in the meantime we can have docs on docs.renovatebot.com instead.

Regardless, these docs should be the first thing you read at https://docs.renovatebot.com/modules/platform/bitbucket/, which is built from https://github.com/renovatebot/renovate/blob/main/lib/modules/platform/bitbucket/readme.md

### PR DESCRIPTION
**Pull request by HonkingGoose**
## Changes

- Add short intro sentence to GitHub and Bitbucket Cloud platform module pages
- Explain that the hosted app is the easiest way to get started

## Context

Closes #29372.

We should point out the easiest way to do things first.

Some users may land on the platform module docs via the `## Supported platform` section of the docs homepage, which directly links to the platform module docs:

```markdown
## Supported Platforms

Renovate works on these platforms:

- [GitHub (.com and Enterprise Server)](./modules/platform/github/index.md)
- [GitLab (.com and CE/EE)](./modules/platform/gitlab/index.md)
- [Bitbucket Cloud](./modules/platform/bitbucket/index.md)
- [Bitbucket Server](./modules/platform/bitbucket-server/index.md)
- [Azure DevOps](./modules/platform/azure/index.md)
- [AWS CodeCommit](./modules/platform/codecommit/index.md)
- [Gitea and Forgejo](./modules/platform/gitea/index.md)
- [Gerrit (experimental)](./modules/platform/gerrit/index.md)
```

If users follow a link like this, and they end up at a platform which has a hosted app, we should:

- point out the easiest way first
- Explain that the info in this docs is _mostly_ meant for self-hosted users. (The issues list for the platform can be relevant to both hosted and self-hosted users).

## Documentation (please check one with an [x])

- [x] I have updated the documentation, or
- [ ] No documentation update is required

## How I've tested my work (please select one)

I have verified these changes via:

- [x] Code inspection only, or
- [ ] Newly added/modified unit tests, or
- [ ] No unit tests but ran on a real repository, or
- [ ] Both unit tests + ran on a real repository

### CODE REVIEW

File: lib/modules/platform/bitbucket/readme.md [hunk 1]
```diff
@@ -1,5 +1,24 @@
 # Bitbucket Cloud
 
+Most of the information on this page is meant for users who want to self-host Renovate on Bitbucket Cloud.
+
+## Easiest way to run Renovate
+
+For most users, the easiest way to get started is to install [the Mend Renovate app for Bitbucket](#).
```
Comment (HonkingGoose) [inline lib/modules/platform/bitbucket/readme.md:7]: Apparently we have a Mend hosted app for Bitbucket. Can you give me the link to the installation page for Bitbucket?

File: lib/modules/platform/bitbucket/readme.md [hunk 2]
```diff
@@ -1,5 +1,24 @@
...
+
+- authenticate the Renovate app to Bitbucket Cloud
+- keep the tokens safe
+- update the Renovate program for you
+
+If you self-host Renovate you must do the things listed above yourself.
+Self-hosting is meant for users with advanced use cases, or who want to be in full control of the bot and the environment it runs in.
+We recommend most users install the Mend Renovate app.
+
+Read the [Security and Permissions](https://docs.renovatebot.com/security-and-permissions/) page to learn about the Security and Permissions needed for the Mend Renovate app.
```
Comment (HonkingGoose) [inline lib/modules/platform/bitbucket/readme.md:18]:
```suggestion
Read the [Security and Permissions](https://docs.renovatebot.com/security-and-permissions/) page to learn about the Security and Permissions needed for the Mend Renovate app.
```

File: lib/modules/platform/github/readme.md [hunk 3]
```diff
@@ -1,5 +1,24 @@
...
+
+- authenticate the Renovate app to GitHub or GitHub Enterprise
+- keep the tokens safe
+- update the Renovate program for you
+
+If you self-host Renovate you must do the things listed above yourself.
+Self-hosting is meant for users with advanced use cases, or who want to be in full control of the bot and the environment it runs in.
+We recommend most users install the Mend Renovate app.
+
+Read the [Security and Permissions](https://docs.renovatebot.com/security-and-permissions/) page to learn about the Security and Permissions needed for the Mend Renovate app.
```
Comment (HonkingGoose) [inline lib/modules/platform/github/readme.md:18]:
```suggestion
Read the [Security and Permissions](https://docs.renovatebot.com/security-and-permissions/) page to learn about the Security and Permissions needed for the Mend Renovate app.
```

File: lib/modules/platform/bitbucket/readme.md (hunk 1 above)
Comment (HonkingGoose) [inline lib/modules/platform/bitbucket/readme.md:7]:
```suggestion
For most users, the easiest way to get started is to install [the Mend Renovate app for Bitbucket](https://marketplace.atlassian.com/apps/1232072/mend?tab=overview&hosting=cloud).
```

File: lib/modules/platform/bitbucket/readme.md [hunk 4]
```diff
@@ -1,5 +1,24 @@
 # Bitbucket Cloud
 
+Most of the information on this page is meant for users who want to self-host Renovate on Bitbucket Cloud.
+
+## Easiest way to run Renovate
+
+For most users, the easiest way to get started is to install [the Mend Renovate app for Bitbucket](https://marketplace.atlassian.com/apps/1232072/mend?tab=overview&hosting=cloud).
```
Comment (rarkins) [inline lib/modules/platform/bitbucket/readme.md:7]:
```suggestion
For most users, the easiest way to get started is to install [the Mend app for Bitbucket](https://marketplace.atlassian.com/apps/1232072/mend?tab=overview&hosting=cloud) and utilize the free Renovate plan.
```

File: lib/modules/platform/bitbucket/readme.md [hunk 5]
```diff
@@ -1,5 +1,24 @@
...
+When you use the app, Mend will:
+
+- authenticate the Renovate app to Bitbucket Cloud
```
Comment (rarkins) [inline lib/modules/platform/bitbucket/readme.md:10]:
```suggestion
- authenticate the app to Bitbucket Cloud
```

File: lib/modules/platform/bitbucket/readme.md [hunk 6]
```diff
@@ -1,5 +1,24 @@
...
+- keep the tokens safe
+- update the Renovate program for you
```
Comment (rarkins) [inline lib/modules/platform/bitbucket/readme.md:12]:
```suggestion
- maintain and update the Renovate version used
```

File: lib/modules/platform/bitbucket/readme.md [hunk 7]
```diff
@@ -1,5 +1,24 @@
...
+
+If you self-host Renovate you must do the things listed above yourself.
+Self-hosting is meant for users with advanced use cases, or who want to be in full control of the bot and the environment it runs in.
+We recommend most users install the Mend Renovate app.
```
Comment (rarkins) [inline lib/modules/platform/bitbucket/readme.md:16]:
```suggestion
We recommend most users install the Mend app.
```

File: lib/modules/platform/bitbucket/readme.md [hunk 8]
```diff
@@ -1,5 +1,24 @@
...
+
+Read the [Security and Permissions](https://docs.renovatebot.com/security-and-permissions/) page to learn about the Security and Permissions needed for the Mend Renovate app.
```
Comment (rarkins) [inline lib/modules/platform/bitbucket/readme.md:18]:
```suggestion
Read the [Security and Permissions](https://docs.renovatebot.com/security-and-permissions/) page to learn about the Security and Permissions needed for the Mend app.
```

File: lib/modules/platform/github/readme.md [hunk 9]
```diff
@@ -1,5 +1,24 @@
 # GitHub and GitHub Enterprise Server
 
+Most of the information on this page is meant for users who want to self-host Renovate on GitHub or GitHub Enterprise Server.
+
+## Easiest way to run Renovate
+
+For most users, the easiest way to get started is to install [the Mend Renovate app](https://github.com/marketplace/renovate) from the GitHub marketplace.
```
Comment (rarkins) [inline lib/modules/platform/github/readme.md:7]:
```suggestion
For users on GitHub Cloud (`github.com`), the easiest way to get started is to install [the Mend Renovate app](https://github.com/marketplace/renovate) from the GitHub marketplace.
```

File: lib/modules/platform/github/readme.md [hunk 10]
```diff
@@ -1,5 +1,24 @@
...
+When you use the app, Mend will:
+
+- authenticate the Renovate app to GitHub or GitHub Enterprise
```
Comment (rarkins) [inline lib/modules/platform/github/readme.md:10]:
```suggestion
- authenticate the Renovate app to GitHub
```

File: lib/modules/platform/github/readme.md [hunk 11]
```diff
@@ -1,5 +1,24 @@
...
+- keep the tokens safe
+- update the Renovate program for you
```
Comment (rarkins) [inline lib/modules/platform/github/readme.md:12]:
```suggestion
- maintain and update the Renovate version used
```

**Review by rarkins — CHANGES_REQUESTED** [review]

File: lib/modules/platform/bitbucket/readme.md (hunk 4 above)
Comment (HonkingGoose) [reply lib/modules/platform/bitbucket/readme.md:7]:
```suggestion
For most users, the easiest way to get started is to install [the Mend app for Bitbucket](https://marketplace.atlassian.com/apps/1232072/mend?tab=overview&hosting=cloud) and use the free Renovate plan.
```

File: lib/modules/platform/bitbucket/readme.md [hunk 12]
```diff
@@ -1,5 +1,24 @@
 # Bitbucket Cloud
 
+Most of the information on this page is meant for users who want to self-host Renovate on Bitbucket Cloud.
+
+## Easiest way to run Renovate
+
+For most users, the easiest way to get started is to install [the Mend app for Bitbucket](https://marketplace.atlassian.com/apps/1232072/mend?tab=overview&hosting=cloud) and use the free Renovate plan.
```
Comment (HonkingGoose) [inline lib/modules/platform/bitbucket/readme.md:7]: [Our style guide](https://github.com/renovatebot/renovate/blob/main/docs/development/style-guide.md) says:

> ## Correct name for the GitHub app
>
> Refer to the GitHub app version of Renovate as "the Mend Renovate app".

I don't know if you still want to follow this style? Your review suggestions all boiled down to `the app`.

Just wanted to check that the review suggestions I just applied are what you really want. :wink:

File: lib/modules/platform/bitbucket/readme.md [hunk 13]
```diff
@@ -1,5 +1,24 @@
 # Bitbucket Cloud
 
+Most of the information on this page is meant for users who want to self-host Renovate on Bitbucket Cloud.
+
+## Easiest way to run Renovate
```
Comment (HonkingGoose) [inline lib/modules/platform/bitbucket/readme.md:5]: Are my changes enough to consider (part) of this issue done?

- #29372

Comment (rarkins) [reply lib/modules/platform/bitbucket/readme.md:5]: Yes!

File: lib/modules/platform/bitbucket/readme.md (hunk 12 above)
Comment (rarkins) [reply lib/modules/platform/bitbucket/readme.md:7]: The Bitbucket Cloud app is a new/different one and not Renovate-specific. The existing guide still is correct for github

**Review by rarkins — DISMISSED** [review]

File: lib/modules/platform/bitbucket/readme.md [hunk 14]
```diff
@@ -1,5 +1,24 @@
...
+
+- authenticate the app to Bitbucket Cloud
+- keep the tokens safe
+- maintain and update the Renovate version used
+
+If you self-host Renovate you must do the things listed above yourself.
+Self-hosting is meant for users with advanced use cases, or who want to be in full control of the bot and the environment it runs in.
+We recommend most users install the Mend app.
+
+Read the [Security and Permissions](./../../../../docs/usage/security-and-permissions.md) page to learn about the Security and Permissions needed for the Mend app.
```
Comment (viceice) [inline lib/modules/platform/bitbucket/readme.md:18]:
```suggestion
Read the [Security and Permissions](../../../security-and-permissions.md) page to learn about the Security and Permissions needed for the Mend app.
```

File: lib/modules/platform/bitbucket/readme.md [hunk 15]
```diff
@@ -1,5 +1,24 @@
...
+
+After you installed the hosted app, please read the [reading list](./../../../../docs/usage/reading-list.md) to learn how to use and configure Renovate.
```
Comment (viceice) [inline lib/modules/platform/bitbucket/readme.md:20]:
```suggestion
After you installed the hosted app, please read the [reading list](../../../reading-list.md) to learn how to use and configure Renovate.
```

File: lib/modules/platform/github/readme.md [hunk 16]
```diff
@@ -1,5 +1,24 @@
...
+- keep the tokens safe
+- maintain and update the Renovate version used
+
+If you self-host Renovate you must do the things listed above yourself.
+Self-hosting is meant for users with advanced use cases, or who want to be in full control of the bot and the environment it runs in.
+We recommend most users install the Mend Renovate app.
+
+Read the [Security and Permissions](./../../../../docs/usage/security-and-permissions.md) page to learn about the Security and Permissions needed for the Mend Renovate app.
+
+After you installed the hosted app, please read the [reading list](./../../../../docs/usage/reading-list.md) to learn how to use and configure Renovate.
```
Comment (viceice) [inline lib/modules/platform/github/readme.md:20]:
```suggestion
After you installed the hosted app, please read the [reading list](../../../reading-list.md) to learn how to use and configure Renovate.
```

**Review by rarkins — APPROVED** [review]

**Pull request merged**